src/protocols/secure_channel/CASESession.cpp

```diff
@@ -198,7 +198,7 @@ CHIP_ERROR CASESession::EstablishSession(SessionManager & sessionManager, Fabric
     ReturnErrorCodeIf(exchangeCtxt == nullptr, CHIP_ERROR_INVALID_ARGUMENT);
     ReturnErrorCodeIf(fabric == nullptr, CHIP_ERROR_INVALID_ARGUMENT);
 
-    err = Init(sessionManager, delegate);
+    err = Init(sessionManager, delegate, ScopedNodeId(peerNodeId, fabric->GetFabricIndex()));
```

Comment:
So the way things are set up, trying to establish a new session to node N as an initiator will pre-emptively evict existing sessions to node N if the table is full?

I guess that makes some senses; if we're trying to establish a session, the existing ones were not usable for some reason....

Reply:
Right now, this eviction hint is not consumed at all (this PR is just setting that up).

The eventual eviction logic will do something a bit more nuanced: if there are sessions on fabrics which are over-minima, those will get reaped first before it attempts to find one on the to-be-established fabric.

We can discuss the specifics of that policy (which I'll detail out clearly in the ensuing PR) and collectively figure out what the right balanced algorithm there should be.